### ISSUE TITLE
[FEATURE] VCC CLI: Allow searching packages by repo ID

### ISSUE THREAD

**Issue by LeafyLuigi** [vrchat-community/creator-companion#531]
I'm frustrated in being unable to search by repo ID when searching for what packages exist in a repo. I'd need to find the URL of each added repo within `.../VRChatCreatorCompanion/Repos/$repo.json`, opening up the file, finding the value for `"url"` and searching using that.

`vpm list repo` should be able to handle an existing added repo ID (such as `com.vrchat.repos.official`) and then searching by the URL held within (in the case of the aforementioned, the url is `https://packages.vrchat.com/official`).

**Comment by superboo07** [comment]
I agree, also not being able to get a list of packages within a repo without reading the json is a very ass backwards design
